packages/app/src/runner/ResizablePanels.vue

```diff
@@ -10,7 +10,7 @@
     @mousemove="handleMousemove"
   >
     <div
-      v-show="showPanel1"
+      v-if="showPanel1"
```

Comment:
Does this has a noticeable impact on presentation speed with really large spec lists? I'm curious if hiding this rather than unmounting it was intentional @marktnoonan

Reply:
Hmmm interesting perspective. I know it's rendering in Run mode as hidden when we really don't need to render it at all.

Reply:
The main reason to keep the `v-show` is that it avoids the state of the list getting blown away when the list is collapsed, since collapsing won't destroy the component. We don't seem to have a cypress-in-cypress test for this behavior, but we probably should add one. Especially for the moment (until the main specs list performs the same way) I am directing people to the inline specs list who like having the search term saved, or their collapsed folders staying collapsed - [for example here](https://github.com/cypress-io/cypress/issues/22263#issuecomment-1155438058). So we should keep that behavior.

There probably is some value for long lists as well, to just render them once and be done with it. 

In run mode, we don't pass anything to this slot, so the extra div itself doesn't seem so painful. But if we do want to remove it, we could wrap the elements related to it (this div and the sibling resize handle) in a `template` and add a new prop to ResizablePanels to control a `v-if` on the `<template>` - mainly to avoid this component having to know about "run" vs "open".

Reply:
fixed: [`e884de5` (#22388)](https://github.com/cypress-io/cypress/pull/22388/commits/e884de51998273e7b7bf1838f677f5002ad6260c)